solvers/bcd_celer.py

```diff
@@ -0,0 +1,342 @@
+from benchopt import BaseSolver, safe_import_context
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from numpy.linalg import norm
+    from numba import njit
+    from mtl_utils.common import sum_squared, _get_dgemm, norm_l21
+
+
+def dual_mtl(alpha, norm_Y2, Theta, Y):
+    n_samples = Y.shape[0]
+    Y /= (n_samples * alpha)
```

Comment:
you are affecting Y outside of the function when you do this

Reply:
so `Y = Y / (n_samples * alpha)` or should I create a copy of Y instead?